examples/java/src/test/java/org/apache/beam/examples/complete/kafkatopubsub/RunKafkaContainer.java

```diff
@@ -0,0 +1,64 @@
+package org.apache.beam.examples.complete.kafkatopubsub;
```

Comment:
How about putting this class under a `utils` package? I guess, `examples/java/src/test/java/org/apache/beam/examples/complete/kafkatopubsub/utils/RunKafkaContainer.java` is a bit more appropriate place.

Reply:
Moved